### PR TITLE
Fix/empty config values

### LINKED ISSUE #406: Traceback if a config file without a `log_level` used
With this config file:

```yaml
# Development UA Client config file. YAML
sso_auth_url: 'https://login.ubuntu.com'
contract_url: 'https://contracts.staging.canonical.com'
data_dir: /tmp/uaclient
log_file: ubuntu-advantage-devel.log
```

I see this traceback:

```pytb
$ UA_CONFIG_FILE=conf2 ua attach
Traceback (most recent call last):
  File "/usr/bin/ua", line 9, in <module>
    load_entry_point('ubuntu-advantage-tools==19.5', 'console_scripts', 'ubuntu-advantage')()
  File "/usr/lib/python3/dist-packages/uaclient/cli.py", line 396, in main
    cfg = config.UAConfig()
  File "/usr/lib/python3/dist-packages/uaclient/config.py", line 54, in __init__
    self.cfg = parse_config()
  File "/usr/lib/python3/dist-packages/uaclient/config.py", line 271, in parse_config
    cfg['log_level'] = cfg['log_level'].upper()
AttributeError: 'int' object has no attribute 'upper'
```

This also occurs if `UA_CONFIG_FILE` points at a non-existent config file (i.e. `$ UA_CONFIG_FILE=nope ua attach`).

### PR DESCRIPTION
**Pull request by powersj**
The default value for logging when it was missing was becoming an
integer (logging.INFO == 20) and then being passed around between
functions that expected a string value for the level.

Rather than storing logging.INFO as the default value for logging, use
the same value/string that we print to the file, namely 'INFO' and then
do the right thing with the string.

Fixes #406

### CODE REVIEW

File: uaclient/cli.py [hunk 1]
```diff
@@ -399,7 +399,7 @@ def main(sys_argv=None):
     args = parser.parse_args(args=cli_arguments)
     cfg = config.UAConfig()
     try:
-        log_level = int(cfg.log_level)
+        log_level = cfg.log_level
```
Comment (OddBloke) [inline uaclient/cli.py]: If we're no longer calling `int()`, nothing here should be raising a `TypeError`; can we drop the try/except here too? (And, presumably, just refer to `cfg.log_level` directly where we're currently using `log_level` in this function?)

Comment (powersj) [reply uaclient/cli.py]: I can catch that, if I catch something like this I was going to suggest using the default value (info). does that work?

Comment (OddBloke) [reply uaclient/cli.py]: So you're suggesting that these lines in `setup_logging`: https://github.com/CanonicalLtd/ubuntu-advantage-client/blob/e74b4c1ad81b1c77a709459d2581a85e1a620c64/uaclient/cli.py#L370-L371 will change to fallback to INFO if we get a `ValueError` from `setLevel`?

That sounds reasonable to me.

**Comment by powersj** [comment]
ready for re-review

**Pull request merged**